### ISSUE TITLE
Clicking on a search field breaks the window on Linux

### ISSUE THREAD

**Issue by MELONumVXA** [godotengine/godot#79820]
### Godot version

4.1.1-stable

### System information

Godot v4.1.1.stable - Nobara Linux 38 (Thirty Eight) - Vulkan (Compatibility) - AMD Radeon RX 570 Series (polaris10, LLVM 15.0.7, DRM 3.52, 6.3.12-204.fsync.fc38.x86_64) () - AMD Ryzen 5 5500 (12 Threads)

### Issue description

Whenever I try to find an button in the Event Configuration window in the Input Map, the window for it minimizes itself and I can't access it anymore unless I restart Godot. On version 4.0.3 provided by the official Fedora repo, the same exact thing happened to the node creation menu when you try to search for a specific node. 

As I were trying to reproduce this bug, I noticed that such thing only happens with the Compatibility renderer.

### Steps to reproduce

1. Create a new project with the Compatibility renderer
2. Open Project > Project Settings > Input Map
3. Add a new action
4. Press the plus sign next to the action to add a new event
5. Click filter inputs, the window gets minimized, it's there, but it's not accessible anymore
![image](https://github.com/godotengine/godot/assets/44049024/244c79a5-2859-4ee9-85e0-eba8f551e4da)

### Minimal reproduction project

N/A

**Comment by Sauermann** [comment]
I am unable to replicate this on Debian X11 Xfce.
Testes with v4.1.stable.official [970459615] and v4.2.dev.custom_build [6588a4a29]

Can you please check if you are using X11 or Wayland? Use `echo $XDG_SESSION_TYPE` to find out.

**Comment by MELONumVXA** [comment]
> I am unable to replicate this on Debian X11 Xfce. Tested with v4.1.stable.official [[9704596](https://github.com/godotengine/godot/commit/970459615f6b2b4151742ec6d7ef8559f87fd5c5)] and v4.2.dev.custom_build [[6588a4a](https://github.com/godotengine/godot/commit/6588a4a29af1621086feac0117d5d4d37af957fd)]
> 
> Can you please check if you are using X11 or Wayland? Use `echo $XDG_SESSION_TYPE` to find out.

Yes, I am using GNOME Wayland, and I have suspicions that it could be the reason I have problems with it. Other pop-up windows would get minimized instantly, but I can access them anyway, unlike the windows in which I click on a search field.

**Comment by Yiannis128** [comment]
This issue is breaking and does not let me use Godot at all. Whenever I click on any field in Godot, the window or sub window will minimize and refuse to come up again. Godot is broken after this until restarted.

**Comment by Calinou** [comment]
This should be fixed in 4.2.dev5 according to https://github.com/godotengine/godot/issues/77333.
